home/private_dot_local/lib/agent-equipment/agent_equipment/authoring.py

```
@@ -0,0 +1,1165 @@
+
+def _target_from_record(value: object) -> str | None:
+    if not isinstance(value, dict):
+        return None
+    harness = value.get("harness")
+    equipment_identity = value.get("equipment_identity")
+    if not isinstance(harness, str) or not isinstance(equipment_identity, str):
+        return None
+    return f"{harness}/{equipment_identity}"
+
+
+def _add_to_documents(
```

Ruff (0.16.1)

[warning] Too many return statements (32 > 6)

(PLR0911)